Test/Integration/TestCase.php

```diff
@@ -44,6 +49,8 @@ public function setUp(): void
     public function tearDown(): void
     {
         $this->clearIndices();
+        $this->algoliaHelper->waitLastTask();
+        $this->clearIndices(); // Remaining replicas
```

Comment:
Could lingering replicas be the reason that some indices were not getting cleaned up earlier?

Reply:
That's what I got in my testing app, with the second clearIndices(), this seems to solve the issue yes (at least when the testing process comes to an end).
I'll try to keep an eye on this.